src/Domain/FaceNormal.cpp

```diff
@@ -10,21 +10,19 @@
 #include "Domain/InterfaceLogicalCoordinates.hpp"
 #include "Domain/Structure/Direction.hpp"  // IWYU pragma: keep
 #include "NumericalAlgorithms/Spectral/Mesh.hpp"
-#include "Utilities/ContainerHelpers.hpp"
 #include "Utilities/GenerateInstantiations.hpp"
 #include "Utilities/Gsl.hpp"
 
 template <size_t VolumeDim, typename TargetFrame>
 void unnormalized_face_normal(
     const gsl::not_null<tnsr::i<DataVector, VolumeDim, TargetFrame>*> result,
-    const Mesh<VolumeDim - 1>& interface_mesh,
+    const Mesh<VolumeDim - 1>& /*interface_mesh*/,
```

Comment:
Could you add a note that this is left over from a refactor and can be deleted?

Reply:
Filed #5177.